www/base/src/app/settings/service/settings.service.coffee

```diff
@@ -0,0 +1,45 @@
+class bbSettings extends Provider
+    constructor: ->
+        @groups = {}
+
+
+    _mergeNewGroup: (oldGroup, newGroup) ->
+        if not newGroup?
+            return undefined
+        if not oldGroup?
+            item.value = item.default_value for item in newGroup.items
+            return newGroup
+        else 
+            for newItem in newGroup.items
+                newItem.value = newItem.default_value
+                for oldItem in oldGroup.items
+                    if newItem.name is oldItem.name and oldItem.value?
+                        newItem.value = oldItem.value
+            return newGroup
+
+    _addSettingsGroup: (group) ->
```

Comment:
_addSettingsGroup can be used in every module config (in the route.coffee). It's not called here because there is no relevant data settings to include here